gno.land/pkg/gnoclient/client_test.go

```diff
@@ -50,3 +50,107 @@ func TestClient_Request(t *testing.T) {
 
 	// XXX: need more test
 }
+
+func TestClient_Call(t *testing.T) {
+	t.Parallel()
+
+	config, _ := integration.TestingNodeConfig(t, gnoenv.RootDir())
+	node, remoteAddr := integration.TestingInMemoryNode(t, log.NewNopLogger(), config)
+	defer node.Stop()
+
+	signer := newInMemorySigner(t, config.TMConfig.ChainID())
+	rpcClient := rpcclient.NewHTTP(remoteAddr, "/websocket")
```

Comment:
You don't actually need to spin up an entire node and keybase to create a client - you can mock this functionality, since they are just interfaces.

These are meant to be unit tests that verify the code is working correctly under specific conditions.

If you want to integrate a live node and a keybase into the testing suite, I suggest you make these types of tests their own category (e2e)

Reply:
I have added mocks as your request, please take a look at [3ecf8f0](https://github.com/gnolang/gno/pull/1565/commits/3ecf8f0836c5443f3c54d3ee4ffe5b387398f754)